packages/twenty-front/src/effect-components/PageChangeEffect.tsx

```diff
@@ -163,10 +167,14 @@ export const PageChangeEffect = () => {
 
   useEffect(() => {
     setTimeout(() => {
-      eventTracker('pageview', {
-        location: {
-          pathname: location.pathname,
-        },
+      setSessionId('testCookie');
```

Comment:
why is this named testCookie?
also `setSettionId` feels strange, I would assume the session is now "testCookie" but that's not what you're doing here? Maybe just initSessionId() / getSessionId - without param? 
cookie name can be stored in a constant SESSION_COOKIE_NAME or something like that